script/Arcueid/Day 2/02_00_ARC02_1D.txt

```diff
@@ -66,11 +66,12 @@ Although there is a difference in the number of things we have accumulated, we s
 [sha:5bb0e0fcbd88c331effae5fa14f52a1c92aa39fd]{
 -- Page 118, Offset 2908.
 -- 熱さがない、夢がない、希望がない、と大人たちは言うけれど、それも仕方のない事だ。
-Adults say that there is no enthusiasm, no dreams, and no hope in us, but that's just the way it is.
+Adults say that we lack enthusiasm, dreams, and hope, but that's just the way it is.
 }
 [sha:4a66c7113921f7c633bcb2bf2d207f930e9c9b80]{
 -- Page 118, Offset 2909.
 -- 今や世界は狭い。小学生ですらこの社会がどれほど行き詰まっているか知っている。
+//absolutely amazing social commentary by nasu wow
```

Comment:
when i first played through arc route, this rant really did not prime me to enjoy the game